lib/src/components/NxProgressBar/__tests__/NxProgressBar.test.tsx

```diff
@@ -4,191 +4,176 @@
+      expect(quickRender({ variant: 'inline' }).queryByText('current progress'))
+          .not.toBeInTheDocument();
     });
 
-    it('removes the .nx-progress-bar__counter when true', function() {
-      expect(getMountedComponent({ showSteps: true })).not.toContainMatchingElement('.nx-progress-bar__counter');
+    it('doesn\'t render when inlineCounter is set to true', function() {
+      expect(quickRender().queryByText('current progress')).toBeInTheDocument();
+      expect(quickRender({ inlineCounter: false }).getByText('current progress')).toBeInTheDocument();
+      expect(quickRender({ inlineCounter: true }).queryByText('current progress'))
+          .not.toBeInTheDocument();
     });
 
-    it('adds a number of .nx-progress-bar__step elements equal to one less than the max when true', function() {
-      expect(getMountedComponent()).not.toContainMatchingElement('.nx-progress-bar__step');
-      expect(getMountedComponent({ showSteps: true }).find('.nx-progress-bar__step').length).toBe(99);
-      expect(getMountedComponent({ showSteps: true, max: 12 }).find('.nx-progress-bar__step').length).toBe(11);
-      expect(getMountedComponent({ showSteps: true, max: 1 }).find('.nx-progress-bar__step').length).toBe(0);
+    it('is overridden by labelSuccess when it is specified and progress reaches 100%', function() {
+      const inProgressEl = renderEl({ labelSuccess: 'complete', value: 50 })!,
+          finishedEl = renderEl({ value: 100 })!,
+          finishedElWithProp = renderEl({ labelSuccess: 'complete', value: 100 })!,
+          finishedElWithPropAndMax = renderEl({ labelSuccess: 'complete', value: 10, max: 10 });
+
+      expect(inProgressEl).toHaveTextContent('current progress');
+      expect(finishedEl).toHaveTextContent('current progress');
+      expect(finishedElWithProp).not.toHaveTextContent('current progress');
+      expect(finishedElWithProp).toHaveTextContent('complete');
+      expect(finishedElWithPropAndMax).not.toHaveTextContent('current progress');
+      expect(finishedElWithPropAndMax).toHaveTextContent('complete');
     });
 
-    it('adds the appropriate modifier class to each step depending on whether it is above, below, or at the ' +
-         'current value', function() {
-      const zeroBar = getMountedComponent({ showSteps: true, max: 12, value: 0 }),
-          partialBar = getMountedComponent({ showSteps: true, max: 12, value: 3 }),
-          fullBar = getMountedComponent({ showSteps: true, max: 12, value: 12 });
+    it('is overriden by labelError when it is specified', function() {
+      const el = renderEl()!,
+          elWithError = renderEl({ labelError: 'oops!' })!;
 
-      expect(zeroBar.find('.nx-progress-bar__step--below-value')).not.toExist();
-      expect(zeroBar.find('.nx-progress-bar__step--at-value')).not.toExist();
-      expect(zeroBar.find('.nx-progress-bar__step--above-value').length).toBe(11);
+      expect(el).toHaveTextContent('current progress');
```

Comment:
Seems unnecessary given the other tests

Reply:
> Seems unnecessary given the other tests

removed here: [4f5d6cf](https://github.com/sonatype/sonatype-react-shared-components/pull/903/commits/4f5d6cf8bfa9484b00e6e80a25b39276491dcfab)